frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -609,7 +609,7 @@ void ProgramStructure::include(cstring filename, cstring ppoptions) {
 void ProgramStructure::loadModel() {
     // This includes in turn core.p4
     std::stringstream versionArg;
-    versionArg << "-DV1MODEL_VERSION=" << V1Model::instance.version;
+    versionArg << "-DV1MODEL_VERSION=20200408";
```

Comment:
It would be better to leave V1Model::instance.version as a static constexpr in the class, rather than hardcoding this value here.